src/query/builder.js

```diff
@@ -960,8 +972,14 @@ assign(Builder.prototype, {
       aggregateDistinct: aggregateDistinct || false
     });
     return this;
-  }
+  },
 
+  // Helper function for clearing or reseting a grouping type from the builder
+  _clearGrouping(grouping){
+    this._statements = reject(this._statements, function(statement){
```

Comment:
Loud and clear,  I also prefer store notation, however I saw function syntax in many places in the apiece, was not sure which was preferred.